aws_lambda.md

```diff
@@ -36,4 +36,24 @@ Then add this to the `plugins` block.
   id 'com.github.johnrengelman.shadow' version '7.1.2'
 ```
 
-Then you can run the `shadowJar` gradle task and create a jar file that has all the code.
\ No newline at end of file
+Then you can run the `shadowJar` gradle task and create a jar file that has all the code.
+
+## Amazon S3
+
+``` groovy
+    implementation 'software.amazon.awssdk:s3:2.20.36'
+```
+``` java
+    public BrickBreakerRequestHandler()
+    {
+        s3Client = S3Client.create();
+    }
+
+```
+
+``` java
+        GetObjectRequest getObjectRequest = GetObjectRequest
+                .builder()
+                .bucket("taub.bbnn").key("trained.json").build();
```

Comment:
Use bucketName and key instead of values specific to your application.
Separate this into multiple lines.